src/components/Layout/Header/Header.tsx

```diff
@@ -79,37 +81,63 @@ export const Header: FunctionComponent<Props> = ({ minimal, colorTheme, navRef }
 
 const HEADER_CONTENT_THEME_CLASS: Record<
     HeaderColorTheme,
-    Record<'container' | 'item' | 'menu' | 'menuItem' | 'menuItemActive' | 'divider' | 'button' | 'panel', string>
+    Record<
+        | 'container'
+        | 'item'
+        | 'menu'
+        | 'menuItem'
+        | 'menuItemActive'
+        | 'divider'
+        | 'button'
+        | 'panel'
+        | 'icon'
+        | 'decoratedText'
+        | 'subText'
+        | 'arrowIcon',
+        string
+    >
 > = {
     white: {
-        container: 'bg-white',
-        item: 'text-black hover:bg-violet-200 focus:ring-black',
-        menu: 'bg-white ring-black',
-        menuItem: 'text-black',
-        menuItemActive: 'bg-violet-200',
+        container: 'bg-gray-50',
+        item: 'text-gray-500 hover:bg-gray-200 focus:ring-black',
+        menu: 'bg-white border-gray-200',
+        menuItem: 'text-gray-700',
+        menuItemActive: 'bg-gray-100',
         divider: 'border-black/25',
         button: 'text-gray-500 hover:bg-violet-200 hover:text-black focus:ring-black',
         panel: 'border-black/25',
+        icon: 'border-black/25 bg-white',
+        decoratedText: 'text-gray-300',
+        subText: 'text-gray-400',
+        arrowIcon: 'bg-white border-gray-200',
     },
     dark: {
-        container: 'bg-black',
-        item: 'text-white hover:bg-violet-600 focus:ring-white',
-        menu: 'bg-gray-700 ring-white',
+        container: 'bg-gray-700',
```

Comment:
Something feels off about this gray-700. I think we're missing a darker shade. Can we add a `gray-800: #14171f` to our Tailwind colors? Then set it as our dark mode container bg for the header and footer? Let's also set the same Gray-800 as the background color for any heroes or pages that had a dark violet background before (Ex: the Resources hero and the Demo/Cody page background). That way it they blend seamlessly with the new nav and footer.